src/event_form/components/price-info/price-info.component.js

```diff
@@ -18,39 +18,19 @@ angular
   });
 
 /* @ngInject */
-function PriceInfoComponent($scope, EventFormData, eventCrud, $rootScope) {
-
-  var controller = this;
-
-  // Price info vars.
-  controller.editPrice = false;
-  controller.priceError = false;
-  controller.invalidPrice = false;
-  controller.savingPrice = false;
-  controller.formPriceSubmitted = false;
-  var originalPrice = [];
-  controller.editingPrice = editingPrice;
-  controller.unsetPriceItemFree = unsetPriceItemFree;
-  controller.setPriceItemFree = setPriceItemFree;
-  controller.deletePriceItem = deletePriceItem;
-  controller.showPriceDelete = showPriceDelete;
-  controller.addPriceItem = addPriceItem;
-  controller.cancelEditPrice = cancelEditPrice;
-  controller.validatePrice = validatePrice;
-  controller.savePrice = savePrice;
+function PriceInfoComponent($scope, $uibModal, EventFormData, eventCrud, $rootScope) {
 
-  init();
+  $scope.setPriceFree = setPriceFree;
+  $scope.openModal = openModal;
+  $scope.savePrice = savePrice;
+  $scope.cancelPrice = cancelPrice;
```

Comment:
`savePrice` en `cancelPrice` zijn niet nodig in de template dus enkel intern beschikbaar stellen. Ook niet via view-model.